### PR TITLE
Make email case insensitive for login

### PR DESCRIPTION
**Pull request by benjefferies**
* Add tests
* Add travis integration

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
```
Comment (lewis12321) [inline .travis.yml]: should that be 3.6.1?? or does that not matter? Im a python noob

File: Pipfile.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (lewis12321) [inline Pipfile.lock]: Dont really know what any of this does?

File: .travis.yml (hunk 1 above)
Comment (benjefferies) [reply .travis.yml]: Ideally it should be 3.6.3 as that's what we specify in the pipfile but shouldn't be a major issue short term. Need to see if we can do that with travos

File: Pipfile.lock (hunk 2 above)
Comment (benjefferies) [reply Pipfile.lock]: Doesn't matter much really but it's a file describing the exact versions of the libraries/dependency's we expect to use when pipenv install

File: .travis.yml (hunk 1 above)
Comment (benjefferies) [reply .travis.yml]: You're right it should have been 3.6.1

**Pull request merged**